cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/registry/AbstractTicketRegistry.java

```diff
@@ -302,4 +304,28 @@ public void setTicketDelegators(@Nullable final List<Pair<Class<? extends Ticket
                                             ticketDelegators) {
         this.ticketDelegators = ticketDelegators;
     }
+
+    /**
+     * Common code to go over expired tickets and clean them up.
+     *
+     * @param ticketStream a stream containing a collection of all tickets in any particular registry holding them.
+     * @param logoutManager the logout manager of any particular ticket registry
+     */
+    protected void cleanupTickets(final Stream<Ticket> ticketStream, final LogoutManager logoutManager) {
```

Comment:
Looks like in both cases you's calling this.getTickets and this.logoutManager. Do these really need to be passed? Can the logout manager also be moved to the abstract ticket registry? So the method signature becomes cleanupTickets() ?